### ISSUE TITLE
ImageNet Pretrained Robust Models : Poor Performance

### ISSUE THREAD

**Issue by iamgroot42** [MadryLab/robustness#28]
I got ~0.3% top-5 accuracy with the L-inf and L-2 robust pretrained models made available.
I have the same pytorch and dill versions given in the README.

**Comment by jobrajac** [comment]
I get the same poor results. Did you find a solution?
@iamgroot42

**Comment by iamgroot42** [comment]
I think I was on a different `dill` version if I remember correctly. It's been a while 😅
